package.json

```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "@docusaurus/module-type-aliases": "^2.4.0",
+    "@docusaurus/plugin-content-docs": "^2.4.1",
```

Comment:
What's this being used for?

Reply:
it's in [sidebar.js](https://github.com/ngrok/ngrok-docs/blob/main/sidebars.js#L14)